app/Http/Controllers/Airlines/StoreAirlineController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers\Airlines;
+
+use App\Http\Requests\StoreAirlineRequest;
+use App\Models\Airline;
+use App\Transformers\AirlineTransformer;
+use Flugg\Responder\Contracts\Responder;
+use Illuminate\Http\JsonResponse;
+
+class StoreAirlineController
+{
+    public function __invoke(StoreAirlineRequest $request, Responder $responder): JsonResponse
+    {
+        $airline = Airline::create([
+            'name' => $request->string('name')->toString(),
+            'description' => $request->string('description')->toString(),
+        ]);
+
+        return $responder->success($airline, new AirlineTransformer())->respond();
```

Comment:
```suggestion
        $airline = Airline::create($request->validated());

        return $responder
            ->success($airline, AirlineTransformer::class)
            ->respond(JsonResponse::HTTP_CREATED);
```
If the request fields match the model's attributes, you can do this, and it'll look much cleaner. `validated()` returns an array with all the inputs you've defined rules for in the Form Request's `rules()`.

Note that I'm responding with the `JsonResponse::HTTP_CREATED` (201) status. This status is for endpoints that create a resource.